docs/Quickstart.md

```diff
@@ -43,8 +43,7 @@ $ heroku create name
 Run these commands to add the `shopify_app` Gem to your app:
```

Comment:
I just noticed that this should also be changed, i.e. "Run this command...".

Reply:
Good catch! Fixed.